conf/v1.routes

```diff
@@ -1 +1,3 @@
 POST       /:nino/:taxYear/sample-endpoint                  v1.controllers.SampleController.handleRequest(nino: String, taxYear: String)
+
+DELETE     /investment/:nino/:taxYear                       v1.controllers.DeleteReliefInvestmentsController.handleRequest(nino: String, taxYear: String)
```

Comment:
should this not be :nino/:taxYear? I think the reliefs/investment part is always there.

Reply:
The context for the api is `individuals/reliefs` it doesn't include `/investment` which is why i put it there as we need to also have in the future a separate part for the reliefs others which will have `other` instead of `investment`

Reply:
In which case we may need to go check some of the other URL's